src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/storage/StorageHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.blobrouter.storage;
+
+import com.azure.storage.blob.BlobServiceClient;
+
+import java.io.ByteArrayInputStream;
+
+public class StorageHelper {
+
+    private StorageHelper() {
+        // utility class
+    }
+
+    public static void uploadFile(
+        BlobServiceClient client,
+        String containerName,
+        String fileName,
+        byte[] fileContent
+    ) {
+        client
+            .getBlobContainerClient(containerName)
+            .getBlobClient(fileName)
+            .getBlockBlobClient()
+            .upload(
+                new ByteArrayInputStream(fileContent), fileContent.length, false
+            )
```

Comment:
not sure if this was made on purpose, but the line fits no problem (87, we allow 120)

Reply:
mistake, will fix

Reply:
done